MIDIchlorians/MIDIchlorians/Assets.xcassets/blueButton.imageset/Contents.json

```diff
@@ -4,6 +4,14 @@
       "idiom" : "universal",
       "filename" : "blueButton.png",
       "scale" : "1x"
+    },
+    {
+      "idiom" : "universal",
```

Comment:
here @advaypal 
this shouldn't be in this diff, if you didn't mean to do this you can discard these changes